pkgs/tools/security/cryptomator/default.nix

```diff
@@ -0,0 +1,113 @@
+{ lib, stdenv, fetchurl, fetchFromGitHub
+, autoPatchelfHook
+, makeDesktopItem
+, fuse, packer
+, maven, openjdk, makeWrapper, glib, wrapGAppsHook
+}:
+
+let
+  pname = "cryptomator";
+  version = "1.5.12";
+
+  src = fetchFromGitHub {
+    owner = "cryptomator";
+    repo = "cryptomator";
+    rev = "${version}";
+    sha256 = "1v7s3y0dh00vbi4zdlg3msv3034sa5qzx5fq5xwi76vs2r37nk2j";
+  };
+
+  desktopItem = makeDesktopItem {
+    name = "cryptomator";
+    comment = "Cloud Storage Encryption Utility";
+    desktopName = "Cryptomator";
+    type = "Application";
+    exec = "cryptomator";
+    icon = "org.cryptomator.Cryptomator";
+    mimeType="application/x-vnd.cryptomator-vault-metadata";
+    categories = "Utility;Security;FileTools;";
+    terminal = "false";
+    extraEntries = ''
+      StartupWMClass=org-cryptomator-launcher-Cryptomator
+    '';
+  };
+
+  # perform fake build to make a fixed-output derivation out of the files downloaded from maven central (120MB)
+  deps = stdenv.mkDerivation {
+    name = "cryptomator-${version}-deps";
+    inherit src;
+
+    buildInputs = [ openjdk maven ];
```

Comment:
Seems like this should be `nativeBuildInputs`.

Reply:
Done